website/client-old/js/services/analyticsServices.js

```diff
@@ -98,6 +98,38 @@
   }
 
   function _gatherUserStats(user, properties) {
+    // filter out animal ears from owned items
+    function _filterAnimalEars (owned) {
+      var animalEarsRegEx = new RegExp('headAccessory_special_[a-z]+Ears', 'g');
+      return Object.keys(owned).reduce(function(animalEars, item) {
+        if (item.match(animalEarsRegEx)) {
+          animalEars.push(item);
+        }
+        return animalEars;
+      }, []);
+    }
+
+    // filter out gem-purchased backgrounds from backgrounds
+    function _filterBackgrounds (owned) {
+      // current free backgrounds
+      var free = ['blue', 'green', 'purple', 'red', 'yellow'];
```

Comment:
The free backgrounds now include `violet` too.

Is it possible to write this code so that it identifies the free backgrounds as being all those in `backgrounds.incentiveBackgrounds` from website/common/script/content/appearance/backgrounds.js ?